grpcutil/carrier.go

```diff
@@ -0,0 +1,40 @@
+package grpcutil
```

Comment:
This file would be more appropriately put into "httpgrpc" package. It has nothing with "gRPC" itself, but works with HTTP requests encoded in protobuf messages (ie. "httpgrpc" package from weaveworks/commons). (I have doubts if this should be in dskit)

Reply:
@pstibrany WDYT, should this file be removed from the PR?

Reply:
> WDYT, should this file be removed from the PR?

Yes, I don't think it belongs to dskit.

Reply:
Removed it.